zerver/openapi/zulip.yaml

```diff
@@ -130,7 +132,9 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/ApiKeyResponse"
+                allOf:
+                  - $ref: "#/components/schemas/ApiKeyResponse"
+                  - description: A typical successful JSON response may look like
```

Comment:
Can we may the string "A typical successful JSON response may look like ..." into a shared string in `components/` or the like?  I think we're likely to want to change it in the future, so avoiding N copies would be really good.

Reply:
@timabbott had changed it already in the next commit(will squash them later) :)